samples/README.md

```diff
@@ -1,5 +1,116 @@
-`Note`:
-- add "include ':samples'" in settings.gradle file.
-- In all the files, replace VAULT_ID, VAULT_URL with the actual vault details in skyflow.Configuration() and TOKEN_URL in TokenProvider implementation
-- Replace CARD_NUMBER_TOKEN,GATEWAY_CVV_GEN_URL,GATEWAY_TOKEN in GenerateCvv.kt file
-- Replace EXPIRATION_DATE_TOKEN,GATEWAY_URL_PULL_FUNDS_URL,TOKEN_LOCAL_URL in PullFunds.kt file.
\ No newline at end of file
+# Android SDK samples
+Test the SDK by adding `VAULT-ID`, `VAULT-URL`, and `SERVICE-ACCOUNT` details in the required places for each sample.
+
+
+## Prerequisites
+- A Skylow account. If you don't have one, register for one on the [Try Skyflow](https://skyflow.com/try-skyflow) page.
+- [Node.js](https://nodejs.org/en/) version 10 or above
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 6.x.x
+- [express.js](http://expressjs.com/en/starter/hello-world.html)
+- Android Gradle plugin 4.2.0 and above
+- Android 5.0 (API level 21) and above
+
+## Prepare
+- Add `include ':samples'` in [settings.gradle](../settings.gradle) file.
+
+### Create the vault
+1. In a browser, navigate to Skyflow Studio.
+2. Create a vault by clicking **Create Vault** > **Upload Vault Schema**.
+3. Choose [data/vaultSchema.json](data/vaultSchema.json).
+4. Once the vault is created, click the gear icon and select **Edit Vault Details**.
+5. Note your **Vault URL** and **Vault ID** values, then click **Cancel**. You'll need these later.
+### Create a service account
+1. In the side navigation click, **IAM** > **Service Accounts** > **New Service Account**.
+2. For Name, enter "SDK Samples". For Roles, choose the required roles for specific action.
+3. Click **Create**. Your browser downloads a **credentials.json** file. Keep this file secure, as you'll need it in the next steps.
+
+### Create a service account bearer token generation endpoint
+1. Create a new directory named `bearer-token-generator`.
+
+        mkdir bearer-token-generator
+2. Navigate to `bearer-token-generator` directory.
+
+        cd bearer-token-generator
+3. Initialize npm
+
+        npm init
+4. Install `skyflow-node`
+
+        npm i skyflow-node
+5. Move the downloaded “credentials.json” file #Create a service account account into the bearer-token-generator directory.        
```

Comment:
Since Android is a client SDK, the user must have an endpoint to get the token,  we have proivded the steps to download a service account and how to generate token from our Node SDK. Also fixed the broken reference to point towards correct step.